src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -147,6 +139,29 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    help='JSON encoded parameters configuration. Use @{file} to load from a file. '
                         'For example: [{"section": "NamingService","parameter": "MaxOperationTimeout"}]')
 
+    with self.argument_context('sf cluster node') as c:
+        c.argument('number_of_nodes_to_add', options_list=['--number-of-nodes-to-add', '--nodes-to-add'], help='number of nodes to add.')
+        c.argument('number_of_nodes_to_remove', options_list=['--number-of-nodes-to-rem', '--nodes-to-remove'], help='number of nodes to remove.')
```

Comment:
Why do you need to rename `--number-of-nodes-to-remove` to `--number-of-nodes-to-rem`,  I personally think that `--number-of-nodes-to-remove` has better readability than `--number-of-nodes-to-rem`